src/main/java/forestry/core/ClimateHandlerServer.java

```diff
@@ -0,0 +1,38 @@
+package forestry.core;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.gameevent.TickEvent;
+import net.minecraftforge.fml.relauncher.Side;
+
+import forestry.api.climate.ClimateManager;
+import forestry.api.climate.IClimateState;
+import forestry.api.climate.IWorldClimateHolder;
+import forestry.core.climate.ClimateStateHelper;
+import forestry.core.network.packets.PacketClimatePlayer;
+import forestry.core.utils.NetworkUtil;
+
+public class ClimateHandlerServer {
+
+	private static IClimateState previousState = ClimateStateHelper.INSTANCE.absent();
+
+	@SubscribeEvent
+	public void onPlayerTick(TickEvent.PlayerTickEvent event) {
+		if (event.phase != TickEvent.Phase.END || event.side != Side.SERVER) {
+			return;
+		}
+		EntityPlayer player = event.player;
+		World world = player.world;
+		BlockPos pos = player.getPosition();
+		IWorldClimateHolder worldClimateHolder = ClimateManager.climateRoot.getWorldClimate(world);
+		IClimateState climateState = worldClimateHolder.getState(pos);
+		if (world.getTotalWorldTime() % 100 == 0
```

Comment:
use `TickHelper` to avoid all the players ticking on the server at the same time